### PR TITLE
Live tiles

### PR DESCRIPTION
**Pull request by IsaacMorris1980**
Live tile initial implementation

### CODE REVIEW

File: Src/MoneyFox.DataAccess/DataServices/PaymentService.cs [hunk 1]
```diff
@@ -129,6 +129,7 @@ public async Task<IEnumerable<Payment>> GetPaymentsByAccountId(int accountId)
                 {
                     var list = await dbContext.Payments
                         .Include(x => x.Category)
+						.Include(x=>x.RecurringPayment)
```
Comment (NPadrutt) [inline Src/MoneyFox.DataAccess/DataServices/PaymentService.cs]: is there a reason for this indent?

File: Src/MoneyFox.Windows/App.xaml.cs [hunk 2]
```diff
@@ -102,6 +102,7 @@ protected override async void OnLaunched(LaunchActivatedEventArgs e)
 
                 BackgroundTaskHelper.Register(typeof(ClearPaymentsTask), new TimeTrigger(60, false));
                 BackgroundTaskHelper.Register(typeof(RecurringPaymentTask), new TimeTrigger(60, false));
+				BackgroundTaskHelper.Register(typeof(LiveTiles), new TimeTrigger(15, false));
```
Comment (NPadrutt) [inline Src/MoneyFox.Windows/App.xaml.cs]: Same here, is there a reason for the difference in indent?

File: Src/MoneyFox.DataAccess/DataServices/PaymentService.cs (hunk 1 above)
Comment (IsaacMorris1980) [reply Src/MoneyFox.DataAccess/DataServices/PaymentService.cs]: probably forgot to convert spaces to tabs or something, will be fixing

File: Src/MoneyFox.Windows/App.xaml.cs (hunk 2 above)
Comment (IsaacMorris1980) [reply Src/MoneyFox.Windows/App.xaml.cs]: same

Comment (IsaacMorris1980) [reply Src/MoneyFox.Windows/App.xaml.cs]: resolved in latest updates

File: Src/MoneyFox.DataAccess/DataServices/PaymentService.cs (hunk 1 above)
Comment (IsaacMorris1980) [reply Src/MoneyFox.DataAccess/DataServices/PaymentService.cs]: resolved in latest updates

**Pull request merged**